### PR TITLE
Add page for API-Portal with main.tf file

### PR DESCRIPTION
**Pull request by abhishek-levo**

### CODE REVIEW

File: docs/api-observability/api-portal.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <a href={window.location.protocol + '//' + window.location.host + '/artifacts/apiportal/main.tf'} download> main.tf </a>
+      )}
+    </BrowserOnly>
+  );
+}
+
+**Levo's API Portal** is a powerful tool for API documentation and insight generation. Here's how it helps developers:
+
+- **Clear API Documentation:** It offers clean, interactive documentation that helps developers understand APIs quickly and effectively.
+- **Interactive API Explorer:** With it's interactive API explorer, developers can explore APIs, make test requests, and understand responses in real-time.
```
Comment (gannaraputeja) [inline docs/api-observability/api-portal.mdx:21]: Please remove make test requests - we do not support this at the moment.

File: docs/api-observability/api-portal.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- **Versioning and Change Management:** The API-Portal simplifies versioning and change management processes, ensuring developers always have access to the latest API documentation and updates.
+
+By leveraging Levo's API Portal, developers can gain deep insights into their APIs and provide exceptional documentation experiences for their users.
+
+### Empower Developers:
+
+- **Centralized access:** Streamline API discovery and exploration with a single source for all your APIs.
+- **Intuitive documentation:** Simplify understanding and usage with clear, interactive documentation for each API.
+- **Efficient integrations:** Enable smooth integration with tools and workflows developers love.
```
Comment (gannaraputeja) [inline docs/api-observability/api-portal.mdx:30]: Let's remove Efficient intuitive documentation, integrations & Seamless testing. Add points below.

Comprehensive documentation: Auto-generated documentation directly from the OpenAPI Schema, providing clear and consistent information about every API endpoint, parameter, and response.
Search and filter: Easily find relevant APIs based on keywords, tags, or categories, saving developers time and effort.
Interactive examples: Try out API calls directly within the portal, testing functionality, and understanding usage patterns.

**Review by gannaraputeja — DISMISSED** [review]
LGTM

**Review by gannaraputeja — APPROVED** [review]
LGTM

File: docs/api-observability/api-portal.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- **Increased developer satisfaction:** Improve developer experience and foster a thriving API community.
+- **Enhanced communication:** Facilitate clear communication between you and your API users.
+
+---------------------------------
+
+## Installation
+
+### Creating a Task Definition in your AWS Account using Terraform.
+
+Download the <DownloadMainTF/> file.
```
Comment (ricekot) [inline docs/api-observability/api-portal.mdx:45]: Why not link directly with a relative path?

**Pull request merged**